images/base/Dockerfile

```diff
@@ -13,5 +13,5 @@ FROM centos:centos7
 RUN yum swap -y -- remove systemd-container\* -- install systemd systemd-libs
 
 RUN yum install -y which git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
-    socat ethtool device-mapper iptables e2fsprogs xfsprogs && \
+    socat ethtool device-mapper iptables && \
```

Comment:
It's ok to leave these here - filesystem utils for XFS and EXT will be generally useful in a number of scenarios